src/Components/ClusterRules/ClusterRules.spec.ct.js

```diff
@@ -19,6 +19,8 @@ import { applyFilters } from '../../../cypress/utils/filters';
 import { cumulativeCombinations } from '../../../cypress/utils/combine';
 import { CHIP_GROUP, CHIP, ROWS } from '../../../cypress/views/filterableTable';
 
+let data = singleClusterPageReport.report.data;
```

Comment:
prefer `const` here since the value is not changed farther

Reply:
done, thanks